drivers/flash/spi_nor.c

```diff
@@ -540,6 +540,45 @@ static int spi_nor_wrsr(const struct device *dev,
 	return ret;
 }
 
+#define MX25R_HIGH_PERF_MODE BIT(1)
+
+#ifdef CONFIG_SPI_NOR_HIGH_PERF_MODE
+static int enable_high_perf_mode(const struct device *dev, uint8_t jedec_id[])
```

Comment:
Can this be named `mx_specific_enable_high_perf_mode`, or `mx_extension_enable_high_perf_mode`?